plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiClassBuilderInterceptor.kt

```diff
@@ -35,8 +25,9 @@ enum class AbiMethodInfo {
 }
 
 sealed class AbiClassInfo {
-    object Public : AbiClassInfo()
-    class Stripped(val methodInfo: Map<Method, AbiMethodInfo>) : AbiClassInfo()
+    object Keep : AbiClassInfo()
+    class Strip(val methodInfo: Map<Method, AbiMethodInfo>) : AbiClassInfo()
```

Comment:
Could you please revert this rename for the time being. We don't usually perform meaningful changes and renames in the same commit, unless strictly necessary.